cloudhub-1/modules/ROOT/Edits/vpn-maintenance.adoc

```diff
@@ -0,0 +1,71 @@
+
+After the October 2022 quarterly maintenance window, Anypoint VPN will no longer support quarterly customer self-upgrades. From this point forward, updates made to Anypoint VPN will adhere to AWS's standard continuous upgrade process. Upgrades will be applied on-demand, with no published maintenance window schedule, no customer opt-in self-upgrade capabilities, with *potential (limited) downtime for customers that do not have a* xref:vpn-high-availability.adoc[High Availability (HA) VPN implementation].
+
+Deploy a xref:vpn-high-availability.adoc[high-availability VPN] solution to avoid any downtime associated with these AWS-delivered security updates. Review the below section for recommendations and troubleshooting.
+
+== Avoiding or Reducing Downtime During an On-demand Maintenance
+
+Mule applications that use Anypoint VPN connectivity are impacted during the on-demand maintenance. The impact on the Anypoint VPN varies depending on the VPN configuration:
+
+* An Anypoint VPN with at least one pair of tunnels configured with BGP routing and DPD (Dead Peer Detection) to reset the tunnels when triggered incurs minimal downtime. The switchover takes a few seconds. See https://help.mulesoft.com/s/article/DPD-Dead-Peer-Detection-and-Anypoint-VPN-Virtual-Private-Networking[this article^] for an explanation of DPD and VPN.
+* An Anypoint VPN tunnel pair configured with Static routing incurs some downtime before reconnection. The reconnection takes longer if you do not have an automated failover mechanism to detect failed tunnels.
+* For an Anypoint VPN configured without BGP routing or a failover mechanism or with only one tunnel configured, the outage is up to 60 minutes long while the update occurs and the original tunnel comes back up.
+
+== Troubleshoot Issues During an Anypoint VPN On-demand Maintenance
+
+These are some of the possible issues you may encounter during the On-demand Maintenance:
+
+=== Tunnels not Coming Up
+
+If the tunnels are not back up within 10-15 minutes at any point in time, try restarting your end of the tunnels to bring them back up.
+
+If your VPN is configured with static routing, try generating 'interesting' traffic towards the Anypoint VPN to bring up the VPN. See https://help.mulesoft.com/s/article/How-to-Generate-Interesting-Traffic-for-Anypoint-VPN[How to Generate Interesting Traffic for Anypoint VPN^] for more information.
+
+=== Tunnel is Going Down Due to Inactivity
+
+If there is no traffic over the VPN tunnel, it goes down due to inactivity. In this case, generate interesting traffic. We recommend you move towards BGP.
```

Comment:
```suggestion
If there is no traffic over the VPN tunnel, the VPN goes down due to inactivity. In this case, generate interesting traffic. We recommend that you move towards BGP.
```

Reply:
Put the word "interesting" in backticks or quotes for consistency with other uses of the elsewhere.